src/pages/RouteDefinitions.tsx

```diff
@@ -178,19 +178,13 @@ export const routes: RouteDefinition[] = [
   }),
   createRouteDefinition({
     path: '/add',
-    nestedPaths: [':currencyIdA', ':currencyIdA/:currencyIdB', ':currencyIdA/:currencyIdB/:feeAmount'],
-    getElement: () => <RedirectDuplicateTokenIds />,
-  }),
-
-  createRouteDefinition({
-    path: '/increase',
     nestedPaths: [
       ':currencyIdA',
       ':currencyIdA/:currencyIdB',
       ':currencyIdA/:currencyIdB/:feeAmount',
       ':currencyIdA/:currencyIdB/:feeAmount/:tokenId',
     ],
-    getElement: () => <AddLiquidity />,
```

Comment:
old AddLiquidity import is no longer used

also I'm assuming RedirectDuplicateTokenIds ends up there, but you might consider renaming it now since the name doesn't communicate info about the page

Reply:
nice catch, and yeah i can rename these